Library/Homebrew/cli/parser.rb

```diff
@@ -614,6 +614,7 @@ def process_option(*args, type:, hidden: false)
         @processed_options.reject! { |existing| existing.second == option.long.first } if option.long.first.present?
         @processed_options << [option.short.first, option.long.first, option.arg, option.desc.first, hidden]
 
+        args.select! { |a| a.start_with?("-") }
```

Comment:
```suggestion
        args.pop # last argument is the description
```

This should also work.